README.md

```diff
@@ -35,12 +44,13 @@ Pay attention to the output from your package manager as the `Datadog CDK Constr
 
 ### AWS CDK
 
-_If you are new to AWS CDK then check out this [workshop][14]._
+- _If you are new to AWS CDK then check out this [workshop][14]._
+- _The following examples assume the use of AWS CDK v2. If you're using CDK v1, import `datadog-cdk-constructs` rather than `datadog-cdk-constructs-v2`._
 
 Add this to your CDK stack:
 
 ```typescript
-import { Datadog } from "datadog-cdk-constructs";
+import { Datadog } from "datadog-cdk-constructs-v2";
 
 const datadog = new Datadog(this, "Datadog", {
   nodeLayerVersion: <LAYER_VERSION>,
```

Comment:
As a sanity check I would try to deploy some sample app's with both dd cdk v1 and v2 to make sure none of the configuration parameters didn't break (they probably didn't but it couldn't hurt to double check if you haven't already).

Reply:
this has been tested!